src/Microsoft.ApplicationInsights.AspNet/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -17,17 +17,12 @@ public static class ApplicationInsightsExtensions
     {
         public static IApplicationBuilder UseApplicationInsightsRequestTelemetry(this IApplicationBuilder app)
```

Comment:
Is that why extension methods are placed in the `Microsoft.Framework.DependencyInjection` namespace? Do we need the term `ApplicationInsights` in the method name because the extension method is defined outside of the `Microsoft.ApplicationInsights` namespace?
Would `UseRequestTrackingMiddleware` be a better method name than `UseApplicationInsightsRequestTelemetry` or is the term `Middleware` implied here by the presence of the `Use` prefix?